actor/v3action/space_test.go

```diff
@@ -220,4 +221,70 @@ var _ = Describe("Space", func() {
 		})
 
 	})
+
+	Describe("GetSpacesByGUIDs", func() {
```

Comment:
See comment above in `application_test.go`.

Reply:
We modeled these tests to be consistent with GetSpaceByName, so I'll change that and GetSpacesByGUIDs tests to be in alignment with your preferences.